lib/util/noop.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var noop = function() {};
+var noop = Function.prototype;
```

Comment:
It seems you have the same mind as @ericherdzik as he showed me that link :dancers: